filebeat/module/elasticsearch/audit/test/test.log-expected.json

```diff
@@ -1,107 +1,107 @@
 [
     {
-        "@timestamp": "2018-06-19T05:16:15,549", 
-        "elasticsearch.audit.event_type": "authentication_failed", 
-        "elasticsearch.audit.layer": "rest", 
-        "elasticsearch.audit.origin_address": "147.107.128.77", 
-        "elasticsearch.audit.principal": "i030648", 
-        "elasticsearch.audit.uri": "/_xpack/security/_authenticate", 
-        "event.dataset": "audit", 
-        "event.module": "elasticsearch", 
-        "input.type": "log", 
-        "log.offset": 0, 
-        "message": "[2018-06-19T05:16:15,549] [rest] [authentication_failed]        origin_address=[147.107.128.77], principal=[i030648], uri=[/_xpack/security/_authenticate]", 
+        "@timestamp": "2018-06-19T05:16:15.549Z",
```

Comment:
I'm surprised by the end line space diff as this file is auto generated there should not be a difference. Did you touch this file manually perhaps?

Reply:
I did manually touch it, to replace the `@timestamp` values in each expected document.

Reply:
Why not generate it?

Reply:
Sure, I can regenerate them. It was a small change that I could make it global search replace so I didn't bother, but I'll regenerate them now.

Reply:
Regenerated in d9a2bac.